node/src/accountant/db_big_integer/big_int_db_processor.rs

```diff
@@ -338,30 +366,36 @@ impl<'a> SQLParamsBuilder<'a> {
     }
 }
 
-struct UniqueKeySpec<'a> {
-    definition_name: &'a str,
-    substitution_name_in_sql: &'a str,
-    value_itself: &'a dyn ExtendedParamsMarker,
+struct TableUniqueKey<'params> {
+    definition_name: &'params str,
+    substitution_name_in_sql: &'params str,
+    value_itself: &'params dyn DisplayableParamValue,
 }
 
-pub enum KnownKeyVariants<'a> {
-    WalletAddress(&'a dyn ExtendedParamsMarker),
-    PendingPayableRowid(&'a dyn ExtendedParamsMarker),
+pub enum KeyVariants<'params> {
+    WalletAddress(&'params dyn DisplayableParamValue),
+    PendingPayableRowid(&'params dyn DisplayableParamValue),
     #[cfg(test)]
     TestKey {
         var_name: &'static str,
         sub_name: &'static str,
-        val: &'a dyn ExtendedParamsMarker,
+        val: &'params dyn DisplayableParamValue,
     },
 }
 
-impl<'a> From<KnownKeyVariants<'a>> for (&'static str, &'static str, &'a dyn ExtendedParamsMarker) {
-    fn from(variant: KnownKeyVariants<'a>) -> Self {
+impl<'params> From<KeyVariants<'params>>
+    for (
+        &'static str,
+        &'static str,
```

Comment:
Instead of using static lifetimes, it would be better if we use the `String` type. That way the reference won't stay for the whole life of the Node and we'll get rid of the lifetimes (at least static).